src/main/java/org/tron/program/Benchmark.java

```diff
@@ -0,0 +1,266 @@
+        defaultMaxTimeRatio * avgDuration / timeBenchmark);
+
+  }
+
+  public static long triggerContractAndReturnDuration(byte[] contractAddress, long feeLimit)
+      throws ContractExeException, ReceiptCheckErrException, VMIllegalException, ContractValidateException {
+    /* ====================================================================== */
+    String params = "0000000000000000000000000000000000000000000000000000000000004e20" +
+        "00000000000000000000000000000000000000000000000000000000000007d0" +
+        "00000000000000000000000000000000000000000000000000000000000000c8";
+    byte[] triggerData = VMTimeBenchmarkUtils
+        .parseABI("timeBenchmark(uint256,uint256,uint256)", params);
+    TVMResult result = VMTimeBenchmarkUtils
+        .triggerContractAndReturnTVMResult(Hex.decode(OWNER_ADDRESS),
+            contractAddress, triggerData, 0, feeLimit, dbManager, null);
+
+    return result.getDuration();
+  }
+
+  public static TimeBenchmarkResult getTimeRatio()
+      throws ContractExeException, ReceiptCheckErrException, VMIllegalException, ContractValidateException, InterruptedException {
+
+    TimeBenchmarkResult result = timeBenchmark();
+
+    return result;
+  }
+
+  public static long getMem() {
+
+    com.sun.management.OperatingSystemMXBean os = (com.sun.management.OperatingSystemMXBean)
+        java.lang.management.ManagementFactory.getOperatingSystemMXBean();
+    long physicalMemorySize = os.getTotalPhysicalMemorySize();
+
+    return (long) (physicalMemorySize / 1024 / 1024 * 0.8);
+  }
+
+  public static boolean checkJavaVersion() {
+    String name = System.getProperty("java.runtime.name").toLowerCase();
+    if (name.indexOf("openjdk") != -1) {
+      return false;
+    }
+    String[] version = System.getProperty("java.vm.specification.version").split("\\.");
+    if (version.length < 2) {
+      return false;
+    } else if (Integer.valueOf(version[0]) * 10000 + Integer.valueOf(version[1]) < 10008) {
+      return false;
+    } else {
+      return true;
+    }
+  }
+
+  public static void main(String[] args) {
+
+    boolean checkVersion = checkJavaVersion();
+
+    if (checkVersion) {
+      System.out.println("1. JAVA VERSION:\nsatisfied");
+    } else {
+      System.out.println("1. JAVA VERSION:\nMUST be oracle jdk, and version >= 1.8");
+    }
+
+    long mem = getMem();
+    System.out.println("2. MEMORY:\nwhen setup java, recommend to use: java -Xmx" + mem + "m");
+
+    initData();
+
+    TimeBenchmarkResult timeResult = null;
+    try {
+      timeResult = getTimeRatio();
+    } catch (ContractExeException e) {
+      e.printStackTrace();
+    } catch (ReceiptCheckErrException e) {
+      e.printStackTrace();
+    } catch (VMIllegalException e) {
+      e.printStackTrace();
+    } catch (ContractValidateException e) {
+      e.printStackTrace();
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1546&issues=AWXr3w8MOlZ5QoPzoUkI&open=AWXr3w8MOlZ5QoPzoUkI)

[ISSUE_KEY]: AWXr3w8MOlZ5QoPzoUkI